src/server/test/web/readingsCompareMeter.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think to be consistent, this file should be named readingsCompareMeterQuantity.js.